extensions/wikia/CommunityPage/models/CommunityPageShortPagesCardModel.class.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class CommunityPageShortPagesCardModel {
+	const SHORT_PAGES_LIMIT = 3;
+
+	/**
+	 * Will return short pages module
+	 * @return array
+	 */
+	public function getData() {
+		$pages = $this->getPages();
+
+		return count( $pages ) > 0 ? [
+			[
+				'type' => 'expand-article',
+				'title' => wfMessage( 'communitypage-cards-expand-articles' )->text(),
+				'icon' => 'expand-article',
+				'pages' => array_slice( $pages, 0, static::SHORT_PAGES_LIMIT ),
+				'fulllistlink' => ( count( $pages ) > static::SHORT_PAGES_LIMIT )
+					? SpecialPage::getTitleFor( 'Shortpages' )->getLocalURL()
+					: ''
```

Comment:
there's a convention to put `?` and `:` at the end of the line: https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#multi-line-statements